spec/lib/active_resource/log_subscriber_spec.rb

```diff
@@ -0,0 +1,39 @@
+class ActiveResourceLogSubscriberTestImplementation < ActiveResource::LogSubscriber
+  attr_reader :output
+
+  def request_without_filtered_logging(input)
+    @output = input
+  end
+end
+
+RSpec.describe ActiveResource::LogSubscriber do
+  subject(:filtered_log) do
+    ActiveResourceLogSubscriberTestImplementation.new.tap { _1.request_with_filtered_logging(event) }.output
```

Comment:
I am not a fan of `_1`, not very readable, but it's ok for now.
Ruby 3.4 introduces the `it` block param (like kotlin), which is pretty cool :)